azure-pipelines.yml

```diff
@@ -32,16 +32,8 @@ steps:
     arguments: '--cache-from $(IMAGE_NAME):latest'
     addDefaultLabels: false
 
-- task: Docker@1
```

Comment:
@rizzkhan1 do we need to do something when removing a task?

Reply:
No. This looks okay